### PR TITLE
Features/8665

### PR DESCRIPTION
**Pull request by sslaws**
## Status
**Ready**

## Fixes or Implements [AB#8665](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/8665)
- [X] Enhancement
- [ ] Bug
- [ ] Documentation

## Description:
Added the ability to add the Authorization header into Swagger.  I found a clean way to implement in Common for all projects.
Upgraded Swagger to latest Swashbuckle package.

The following changes were made in addition to the core work:
- Moved Hangfire reference for WebClient and AdminWebClient to common and removed associated packages.
- Removed AdminWebClient and WebClient.xml files and added them to ignore files.
- Added missing AllowAnonymous annotation to UserProfile controller for termsofservice 

## Testing
- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [X] Not Required

### Steps to Reproduce:
If this is a bug, please provide details on how to reproduce the code.

### UI Changes
Yes to Swagger UI

### Browsers Tested
- [X] Chrome
- [ ] Safari
- [ ] Edge
- [ ] Firefox

Provide an explanation for any test(s) not completed.
Manually performed Swagger tests as we don't have any functional tests.

## Notes/Additional Comments
None

### CODE REVIEW

File: Apps/WebClient/src/Server/Controllers/BetaRequestController.cs [hunk 1]
```diff
@@ -57,15 +53,17 @@ public class BetaRequestController
...
         /// <response code="200">The beta request record was saved.</response>
         /// <response code="400">The beta request was already inserted.</response>
         /// <response code="401">The client must authenticate itself to get the requested response.</response>
         /// <response code="403">The client does not have access rights to the content; that is, it is unauthorized, so the server is refusing to give the requested resource. Unlike 401, the client's identity is known to the server.</response>
         [HttpPut]
         [Authorize(Policy = UserPolicy.UserOnly)]
-        public IActionResult CreateBetaRequest([FromBody] BetaRequest betaRequest)
+        public IActionResult CreateBetaRequest([FromHeader] string authorization, [FromBody] BetaRequest betaRequest)
         {
+            // this.logger.logTrace(authorization);
```
Comment (leobcgov) [inline Apps/WebClient/src/Server/Controllers/BetaRequestController.cs]: what is our best practice re. logging inputs in overall of this project? Should we log all the inputs at the beginning of each method?

File: Apps/AdminWebClient/src/AdminWebClient.csproj [hunk 2]
```diff
@@ -78,7 +76,7 @@
     <!-- Include the newly-built files in the publish output -->
     <ItemGroup>
       <DistFiles Include="$(SpaRoot)dist\**" />
-      <ResolvedFileToPublish Include="@(DistFiles-&gt;'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
+      <ResolvedFileToPublish Include="@(DistFiles->'%(FullPath)')" Exclude="@(ResolvedFileToPublish)">
```
Comment (leobcgov) [inline Apps/AdminWebClient/src/AdminWebClient.csproj:79]: Can this '>' symbol cause issue to our devs who use mac?

File: Apps/WebClient/src/Server/Controllers/BetaRequestController.cs (hunk 1 above)
Comment (sslaws) [reply Apps/WebClient/src/Server/Controllers/BetaRequestController.cs]: We don’t have logging in many of our controllers and we haven’t DJs used a standard.

File: Apps/AdminWebClient/src/AdminWebClient.csproj (hunk 2 above)
Comment (sslaws) [reply Apps/AdminWebClient/src/AdminWebClient.csproj:79]: VS on Windows encodes it this way while VS on Mac does the &gt way.

**Pull request merged**